src/functions.js

```diff
@@ -7,6 +7,37 @@ import {
   createTodoForm, closeModal, projectForm, displayTasks,
 } from './DOM';
 
+function sortDates(array) {
+  return array.sort((a, b) => {
+    if (a.duedate < b.duedate) {
+      return -1;
+    }
+    if (a.duedate > b.duedate) {
+      return 1;
+    }
+    return 0;
+  });
+}
+
+function orderTodoArray(todoArray) {
```

Comment:
- [ ] [REQUIRED] You should be using arrow functions instead of the `function` keyword
![image](https://user-images.githubusercontent.com/73254040/100157654-822af400-2e70-11eb-877c-11baeab18e07.png)
[Source](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622602/)